app/rest/openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -36,6 +36,8 @@
 public class OpenShiftServiceImpl implements OpenShiftService {
     private static final Logger LOGGER = LoggerFactory.getLogger(OpenShiftServiceImpl.class);
 
+    private static final String OPENSHIFT_PREFIX = "intg-";
```

Comment:
Can we discuss on the prefix ;-) ? is 'intg-' a common acronym for "integrations" ? (sorry when I'm ignorant here"). What about just to use "i-" as the names themselves can be also large.

Reply:
yeah, I was thinking of just using `s-` but then Keith opened issue about pod names and ended up with `intg-`, let me change it to just `i-`